src/main/java/com/redhat/exhort/providers/BaseJavaProvider.java

```diff
@@ -25,14 +25,16 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.TreeMap;
+import org.cyclonedx.model.Component;
 
 public abstract class BaseJavaProvider extends Provider {
 
   protected BaseJavaProvider(Ecosystem.Type ecosystem, Path manifest) {
     super(ecosystem, manifest);
   }
 
-  void parseDependencyTree(String src, int srcDepth, String[] lines, Sbom sbom) {
+  void parseDependencyTree(
```

Comment:
I would rather not expose the CycloneDX data model outside of the CycloneDX parser. You should either use constants or a custom `Enum`

Reply:
Ok, it's not exposed now. Added a conversion step in CycloneDXSbom.java